libparsec/crates/platform_device_loader/src/lib.rs

```diff
@@ -53,6 +67,8 @@ pub enum LoadDeviceError {
     InvalidData,
     #[error("Failed to decrypt file content")]
     DecryptionFailed,
+    #[error(transparent)]
+    Keyring(#[from] keyring::Error),
```

Comment:
What does this error means ?

I have the feeling we should instead have both a generic errors and some useful errors:
- `Internal(#[from] anyhow::Error` that wraps any unknown error
- `KeyringNotAvailable`, `NoCorrespondingEntryInKeyring`, etc.